tests/common/test_version.py

```diff
@@ -113,39 +111,43 @@ def test_distro_is_correct_for_exception_case(self, *args): # pylint: disable=us
             osinfo = get_distro()
 
         self.assertListEqual(default_list, osinfo)
-        return
+
+    def test_lis_version(self):
```

Comment:
for new tests we are following this as naming convention: test_<subject_under_test>_should_<behaviour_being_tested>

this test could be named "test_get_lis_version_should_return_a_string"